src/test/java/api/items/MarkItemInProcessApiTests.java

```diff
@@ -0,0 +1,198 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.net.MalformedURLException;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static api.support.InstanceSamples.smallAngryPlanet;
+import static org.folio.inventory.domain.items.CirculationNote.*;
+import static org.folio.inventory.domain.items.Item.CIRCULATION_NOTES_KEY;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.*;
+import static support.matchers.ItemMatchers.isInProcess;
+import static support.matchers.RequestMatchers.hasStatus;
+import static support.matchers.RequestMatchers.isOpenNotYetFilled;
+import static support.matchers.ResponseMatchers.hasValidationError;
+
+@RunWith(JUnitParamsRunner.class)
+public class MarkItemInProcessApiTests extends ApiTests {
+  private IndividualResource holdingsRecord;
+
+  @Before
+  public void createInstanceAndHoldingsRecord() throws Exception {
+    final IndividualResource instance = instancesClient
+      .create(smallAngryPlanet(UUID.randomUUID()));
+
+    holdingsRecord = holdingsStorageClient.create(new HoldingRequestBuilder()
+      .forInstance(instance.getId()));
+
+  }
+
+  @Test
+  public void testMarkItemInProcessCirculationNotes() throws Exception {
+    final IndividualResource createdItem = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingsRecord.getId())
+      .withCheckInNote()
+      .canCirculate());
+
+    final var itemMarkedAsTargetStatus = markItemInProcess(createdItem).getJson();
+    final var itemCirculationNotes = itemMarkedAsTargetStatus.getJsonArray(CIRCULATION_NOTES_KEY);
+    final var checkInNote = itemCirculationNotes.getJsonObject(0);
+
+    assertThat(checkInNote.getString(NOTE_TYPE_KEY), is("Check in"));
+    assertThat(checkInNote.getString(NOTE_KEY), is("Please read this note before checking in the item"));
+    assertThat(checkInNote.getBoolean(STAFF_ONLY_KEY), is(false));
+    assertFalse(false);
+  }
+  @Parameters({
+    "Available"
+    ,"Awaiting delivery"
+    ,"In transit"
+    ,"Lost and paid"
+    ,"Missing"
+    ,"Order closed"
+    ,"Paged"
+    ,"Withdrawn"
+  })
+  @Test
+  public void canMarkItemInProcessWhenInAllowedStatus(String initialStatus) throws Exception {
+    final IndividualResource createdItem = itemsClient.create(new ItemRequestBuilder()
+      .forHolding(holdingsRecord.getId())
+      .withStatus(initialStatus)
+      .canCirculate());
+
+    assertThat(markItemInProcess(createdItem).getJson(), isInProcess());
+    assertThat(itemsClient.getById(createdItem.getId()).getJson(), isInProcess());
+  }
+
+  @Parameters({
+    "Aged to lost"
+    ,"Checked out"
+    ,"Claimed returned"
+    ,"Declared lost"
+    ,"In process"
+    ,"In process (non-requestable)"
+    ,"Intellectual item"
+    ,"Long missing"
+    ,"On order"
+    ,"Restricted"
+    ,"Unavailable"
+    ,"Unknown"
+  })
+
+  @Test()
```

Comment:
Remove unnecessary parentheses.

Reply:
What parenthess you consider unnecessary?

Reply:
```suggestion
  @Test
```

The ones after `@Test`